src/main/java/org/broadinstitute/hellbender/tools/genome/SparkGenomeReadCounts.java

```diff
@@ -42,7 +43,7 @@
     protected static final String DROP_NON_AUTOSOMES_SHORT_NAME = "noxy";
     protected static final String DROP_NON_AUTOSOMES_LONG_NAME = "noXYMT";
 
-    @Argument(doc = "Remove X, Y and MT regions",
+    @Argument(doc = "Remove X, Y, GL*, NC_*, and MT regions, regardless of intervals specified.  Use -L (or -XL) and disable this option for exact specification of intervals.  This option may be removed in the future.",
```

Comment:
It's weird to have a boolean flag that defaults to true.   It would probably be better to invert this.   The change to the documentation looks good though.